docs/install/core-install.rst

```diff
@@ -201,6 +201,12 @@ your operating system's native package manager using the following commands.
 See :doc:`rocm-install-on-linux:index` for guidance on installing the ROCm
 software stack.
 
+.. note::
+
+   Omniperf will use the first version of ``Python3`` found in your system's
+   ``PATH``. If the default version of Python3 is older than 3.8, you may need to
+   update your system's ``PATH`` to point to a newer version of Python3.
```

Comment:
Maybe add a note saying: If you wish to always run Omniperf with a particular version of python, we recommend (e.g.,) creating a bash alias that invokes:

```
omniperf-mypython="/path/to/my/python3 /path/to/omniperf"
```

Reply:
or something like this

Reply:
That's a good idea. I think that we can move these notes to the "core installation" section. They are applicable to all installation methods.